Source/Core/AudioCommon/WaveFile.cpp

```diff
@@ -159,9 +161,9 @@ void WaveFileWriter::AddStereoSamplesBE(const short* sample_data, u32 count,
   {
     Stop();
     file_index++;
-    std::ostringstream filename;
-    filename << File::GetUserPath(D_DUMPAUDIO_IDX) << basename << file_index << ".wav";
-    Start(filename.str(), sample_rate_divisor);
+    const std::string filename =
+        fmt::format("{:s}{:s}{:d}.wav", File::GetUserPath(D_DUMPAUDIO_IDX), basename, file_index);
```

Comment:
Any particular reason you're using format specifiers for trivial cases such as this?

Reply:
I'd be curious about this too.  It's done in multiple PRs.  I'd say it hurts readability.  Generally, when I see formatting codes, I expect some special handling.  Now I have to look at each one to see what is going on.

Reply:
Sorry, I usually default using to format specifiers, but have been noticing a trend against it around in the codebase.  I can change it to the default "{}" if that's wanted.

Reply:
Though I disagree that being more explicit hurts readability, I guess it's not my codebase to care.

Reply:
I can see the argument for either side but since we generally don't use them in Dolphin I agree that it's confusing to see them here.